### ISSUE TITLE
Unicode Symbol error

### ISSUE THREAD

**Issue by ReDss-TS** [josdejong/svelte-jsoneditor#149]
Have an error "Expected ',' or '}' after property value in JSON", 
but JSON has Unicode Symbol

````
{
    "test": {
        "en": [
                {
                    "l": "\u0022test1\u0022",
                    "k": "\u0022test1\u0022"
                },
                {
                    "l": "test2",
                    "k": "test2"
                }
            ]
    }
}
````

**Comment by josdejong** [comment]
Thanks for reporting, I can reproduce this issue. Looks like something weird is going on with escaping of the unicode characters.

**Comment by josdejong** [comment]
It turns out that both double quote `"` and unicode characters of control characters (like double quote `\u0022` or line feed `\u000A`) where not correctly escaped. This is fixed now in `v0.7.6`.

Note that as soon as you do something with a JSON document containing `\u0022` unicode characters, they will be replaced 
by `\"` which is the same.
